designer/server/src/test/scala/pl/touk/nussknacker/ui/component/ComponentsUsageHelperTest.scala

```diff
@@ -168,7 +168,12 @@ class ComponentsUsageHelperTest extends AnyFunSuite with Matchers with TableDriv
     )
 
     forAll(table) { (processesDetails, expected) =>
+      import pl.touk.nussknacker.engine.util.Implicits._
+
       val result = ComponentsUsageHelper.computeComponentsUsage(defaultComponentIdProvider, withComponentsUsages(processesDetails))
+        .mapValuesNow(_.map {
+          case (baseProcessDetails, nodeIds) => (baseProcessDetails.mapProcess(_ => ()), nodeIds.map(_.nodeId))
```

Comment:
I think that we shouldn't do _.nodeId extraction. Instead we should change expectations to make sure that everything works as expected

Reply:
Done :)